src/ui/CryptoUi.java

```diff
@@ -1,7 +1,58 @@
 package ui;
 
+import domain.CaesarsStrategy;
+import domain.CryptoContext;
+import domain.CryptoStrategy;
+import domain.ReflectionStrategy;
+
+import javax.swing.*;
+
 public class CryptoUi {
+    public void CryptoContext(){
```

Comment:
I think there was a mistake in UML - this method is not necessary,maybe it was supposed to be a CryptoUI constructor. And CryptoContext should be defined as a private variable in this class and initialized in the CryptoUI constructor.